ci-operator/config/terraform-redhat/terraform-provider-ocm/terraform-redhat-terraform-provider-ocm-main__images.yaml

```diff
@@ -0,0 +1,30 @@
+binary_build_commands: make install
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+images:
+- dockerfile_path: build/ci-tf-e2e.Dockerfile
```

Comment:
isn't it already being built [here](https://github.com/openshift/release/blob/f7ac3ee42a5bdb79becfb118ac5c8baa98e8acbb/ci-operator/config/terraform-redhat/terraform-provider-ocm/terraform-redhat-terraform-provider-ocm-main.yaml#L8)?
I think we just need to add the ``promotion`` stanza there

Reply:
You are right. Adding it to the existing one. Thank you.